### DISCUSSION TITLE
Unable to add Ring Intercom as "device"

### DISCUSSION THREAD

**Discussion by tiarex86** [tsightler/ring-mqtt#667] [Support Q&A]
Hi, I've already setup a Ring Intercom via its mobile app and everything works fine.
Now I'm going to integrate it into HA, installed Ring-MQTT and successfully configured, in fact it founds my device as you can see below (extracted from Ring-MQTT logs):
```
2023-07-11T10:37:09.214Z ring-attr [Ingresso] ring/711d09cc-1301-4992-a849-5f9f9ea2b583/intercom/343ea4b09703/info/state {"batteryLevel":58,"firmwareStatus":"Up to Date","lastUpdate":"2023-07-11T01:36:24Z","wirelessNetwork":"xxx","wirelessSignal":-53}
2023-07-11T10:37:09.214Z ring-attr [Ingresso] ring/711d09cc-1301-4992-a849-5f9f9ea2b583/intercom/343ea4b09703/battery/attributes {"batteryLevel":58}
2023-07-11T10:37:09.215Z ring-attr [Ingresso] ring/711d09cc-1301-4992-a849-5f9f9ea2b583/intercom/343ea4b09703/wireless/attributes {"wirelessNetwork":"xxx","wirelessSignal":-53}
2023-07-11T10:37:34.834Z ring-disc Republishing existing device id: 343ea4b09703
2023-07-11T10:37:34.834Z ring-disc HASS config topic: homeassistant/sensor/711d09cc-1301-4992-a849-5f9f9ea2b583/343ea4b09703_battery/config
2023-07-11T10:37:34.835Z ring-disc {
  name: 'Ingresso Battery',
  unique_id: '343ea4b09703_battery',
  state_topic: 'ring/711d09cc-1301-4992-a849-5f9f9ea2b583/intercom/343ea4b09703/info/state',
  device_class: 'battery',
  unit_of_measurement: '%',
  state_class: 'measurement',
  value_template: '{{ value_json["batteryLevel"] | default("") }}',
  json_attributes_topic: 'ring/711d09cc-1301-4992-a849-5f9f9ea2b583/intercom/343ea4b09703/battery/attributes',
  availability_topic: 'ring/711d09cc-1301-4992-a849-5f9f9ea2b583/intercom/343ea4b09703/status',
  payload_available: 'online',
  payload_not_available: 'offline',
  device: {
    ids: [ '343ea4b09703' ],
    name: 'Ingresso',
    mf: 'Ring',
    mdl: 'Intercom'
  }
}
```

Unfortunately no Ring device appears on HA.
I also changed Ring-MQTT configuration from **branch: "blank"** to **branch: dev** without success.
Any thought about how to solve it?
Thanks for any kind of support!

**Comment by tsightler** [comment]
Unfortunately it is impossible to troubleshoot issues with only log snippets, full logs are required.  For example, above are only part of what should be published for an intercom but none of the discovery data during initial startup is in the snippet so I can't really tell what is going on.  Please open a support issue and provide full logs and I can try to help.  Or I can convert this conversation to an issue if you prefer, but either way full logs will be required.

As a general rule, if you are seeing discovery messages in the logs, but nothing is showing up in Home Assistant, then the Home Assistant Mosquitto integration is not working, but this is a generalization and not possible to tell with just a snippet.

**Reply by tiarex86** [reply]
Thanks for answering me.
Yes please, if you can convert it to a support issue (never done before).
Could you also point me to HA folder where I can find Ring-MQTT logs?
Thanks a lot.

**Reply by tsightler** [reply]
How to get full docker logs for the addon is documented here:
https://github.com/tsightler/ring-mqtt-ha-addon/blob/main/GET-LOGS.md

Issue https://github.com/tsightler/ring-mqtt/issues/668 has been opened, please continue any discussion there.  Thanks!
